### PR TITLE
Add linux-arm64 to releases

### LINKED ISSUE #352: Create linux-arm64 binaries
For linux-arm users along with supporting:

* https://github.com/vmware-tanzu/carvel-kapp-controller/issues/573

### PR DESCRIPTION
**Pull request by benmoss**
Fixes https://github.com/vmware-tanzu/carvel-imgpkg/issues/352

### CODE REVIEW

**Review by joaopapereira — CHANGES_REQUESTED** [review]
There is another place that needs some change.

the step `get-checksums-from-draft-release` when we are retrieving the checksums, needs to change to include the checksum of the arm version.
Also the `hack/build-binaries.sh` script also need to manually create the arm version so that we can validate the output of the Github action step.

I think that should be it

**Comment by benmoss** [comment]
@joaopapereira updated!

File: hack/build-binaries.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ LDFLAGS="-X github.com/vmware-tanzu/carvel-imgpkg/pkg/imgpkg/cmd.Version=$VERSIO
 GOOS=darwin GOARCH=amd64 go build -ldflags="$LDFLAGS" -trimpath -o imgpkg-darwin-amd64 ./cmd/imgpkg/...
 GOOS=darwin GOARCH=arm64 go build -ldflags="$LDFLAGS" -trimpath -o imgpkg-darwin-arm64 ./cmd/imgpkg/...
 GOOS=linux GOARCH=amd64 go build -ldflags="$LDFLAGS" -trimpath -o imgpkg-linux-amd64 ./cmd/imgpkg/...
+GOOS=linux GOARCH=arm64 go build -ldflags="$LDFLAGS" -trimpath -o imgpkg-linux-arm64 ./cmd/imgpkg/...
 GOOS=windows GOARCH=amd64 go build -ldflags="$LDFLAGS" -trimpath -o imgpkg-windows-amd64.exe ./cmd/imgpkg/...
 
 shasum -a 256 ./imgpkg-darwin-amd64 ./imgpkg-darwin-arm64 ./imgpkg-linux-amd64 ./imgpkg-windows-amd64.exe
```
Comment (joaopapereira) [inline hack/build-binaries.sh]: It is missing the arm shasum in this line

Comment (benmoss) [reply hack/build-binaries.sh]: Good catch

**Review by joaopapereira — APPROVED** [review]
LGTM, we should in the future try to get this more automated in some way without having to change so many places

**Pull request merged**